cypress/integration/operator.js

```diff
@@ -0,0 +1,91 @@
+
+describe('Operator creation basic tests', function () {
+    const testOp1 = {
+        businessId: "1234567-9",
+        name: "e2eAcmeCorp",
+        addrBillingStreet: "Billing Street",
+        addrBillingPostalCode: "999999",
+        addrBillingCity: "Megalopolis",
+        addrVisitStreet: "Visiting Street",
+        addrVisitPostalCode: "000000",
+        addrVisitCity: "Hyperpolis",
+        mobilePhone: "123 12345678",
+        telephone: "555 4567890",
+        email: "e2eacmecorp@localhost",
+        web: "www.acmecorp.acme"
+    };
+
+    // Login in only once before tests run
+    before(function () {
+        cy.login();
+    });
+
+    beforeEach(function () {
+        // Session cookies will not be cleared before the NEXT test starts
+        cy.preserveSessionOnce();
+
+        cy.visit('/#/own-services');
+    });
+
+    it('should navigate to own services', function () {
+        cy.contains('Omat palvelutiedot');
+    });
+
+    it('should validate and invalidate business id', function () {
+
+        // Uncomment these when YTJ service is stubbed and creation is tested fully
+        // cy.get('#select-operator-at-own-services').click();
+        // cy.contains(testOp1.name).should('not.exist');
+        // cy.contains('Omat palvelutiedot').click();
+
+        cy.get('#btn-add-new-transport-operator').click();
+        cy.get('#btn-submit-business-id').as('btnSubmit')
+        cy.get('@btnSubmit').should('be.disabled')
+        cy.get('#input-business-id').type('1')
```

Comment:
admirable systematic approach, but you could add 1234567- with one line and check disabled status.

Reply:
Yes, but the world would be less perfect. 
Let's do  like that when execution time or case length are too long :)